cmake/LibDataChannelConfig.cmake

```diff
@@ -0,0 +1,5 @@
+include("${CMAKE_CURRENT_LIST_DIR}/LibDataChannelTargets.cmake")
+
+# For forward compatibility
+add_library(LibDataChannel::datachannel ALIAS LibDataChannel::LibDataChannel)
```

Comment:
The alias target don't export. so that, the application use find_package() that don't import it!

Reply:
Could you describe the failing setup? It works fine on my side after installation with:
```
cmake_minimum_required(VERSION 3.7)
project(test-libdatachannel)

find_package(Threads REQUIRED)
find_package(LibDataChannel REQUIRED)

add_executable(test main.cpp)
set_target_properties(test PROPERTIES CXX_STANDARD 17)
target_link_libraries(test Threads::Threads LibDataChannel::datachannel)
```

Reply:
It is ok!